internal/model/request.go

```diff
@@ -1,12 +1,16 @@
 package model
 
+import "time"
+
 type PingRequest struct {
 	Data string `json:"data" validate:"required"`
 }
 type SignUpCredentials struct {
-	Name     string `json:"name" binding:"required"`
-	Email    string `json:"email" binding:"required"`
-	Password string `json:"password" binding:"required"`
+	Name                  string    `json:"name" binding:"required"`
```

Comment:
Don't think binding tag is used. Instead use the `validate` tag as per this

> Use github.com/PereRohit/util/validator to validate the struct fields by adding the validate struct tags. This should validate the required and specific format fields without the need for every field validation manually.
> 
> This validation should be done at the handler level.

Reply:
done